packages/core/src/RenderPipeline/PipelineUtils.ts

```diff
@@ -155,5 +209,12 @@ export class PipelineUtils {
     );
 
     rhi.drawPrimitive(blitMesh._primitive, blitMesh.subMesh, program);
+
+    if (renderBufferStoreAction === RenderBufferStoreAction.BlitMSAA) {
+      destination?._blitRenderTarget();
+    }
+
+    // Revert
+    context.flipProjection = originalFlipProjection;
```

Comment:
Why do this? The strategy should be that whoever consumes should make corrections

Reply:
done